BootloaderCorePkg/Tools/SblFwUpdate.py

```diff
@@ -0,0 +1,32 @@
+## @ SblFwUpdate.py
+#
+# Copyright (c) 2017-2020, Intel Corporation. All rights reserved.<BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##

```

Comment:
May need a couple of lines of documentation about the intended usage and where it may need customization.

Reply:
done